specification/security/resource-manager/Microsoft.Security/preview/2021-01-01-preview/secureScore.json

```diff
@@ -0,0 +1,603 @@
+            "$ref": "./examples/secureScores/ListSecureScoreControlsForNameWithExpand_managementGroup_builtin_example.json"
+          }
+        },
+        "tags": [
+          "Secure Score"
+        ],
+        "description": "Get all security controls for a specific initiative within a scope",
+        "operationId": "SecureScoreControls_ListBySecureScore",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/secureScoreName"
+          },
+          {
+            "$ref": "#/parameters/ExpandControlsDefinition"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecureScoreControlList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Security/secureScoreControls": {
+      "get": {
+        "x-ms-examples": {
+          "List all secure scores controls (Subscription)": {
+            "$ref": "./examples/secureScores/ListSecureScoreControls_subscription_example.json"
+          },
+          "List all secure scores controls (Management Group)": {
+            "$ref": "./examples/secureScores/ListSecureScoreControls_managementGroup_example.json"
+          }
+        },
+        "tags": [
+          "Secure Score Controls"
+        ],
+        "description": "Get all security controls within a scope",
+        "operationId": "SecureScoreControls_List",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/ExpandControlsDefinition"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecureScoreControlList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Security/secureScoreControlDefinitions": {
```

Comment:
you have removed the corresponding subscription level api (/subscriptions/{subscriptionId}/providers/Microsoft.Security/secureScoreControlDefinitions) - if I understand correctly, since that api exists in the last stable version (2020-01-01) it cannot be removed in a subsequent preview api version

Reply:
please add it back, or if you really want to remove it - it must go through breaking change approval - add the BreakingChangesReviewRequired label.

Reply:
Added back.
Thanks for notice